src/cuga/backend/cuga_graph/nodes/cuga_flow/README.md

```
@@ -0,0 +1,286 @@
+cuga start flow_agent_inline loan_approval
+cuga start flow_agent_inline receive_order
+cuga start flow_agent_inline trip_planner
+```
+
+Each app directory follows the same layout: a BPMN file, a `flow_config.yaml` referencing it, agent definitions, and per-task/gateway policy markdown files under `policies/`.
+
+---
+
+## Module Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)